src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2754,7 +2754,7 @@ private unsafe void SendManifest(byte[]? rawManifest)
 
         // Helper to deal with the fact that the type we are reflecting over might be loaded in the ReflectionOnly context.
         // When that is the case, we have the build the custom assemblies on a member by hand.
-        internal static Attribute? GetCustomAttributeHelper(
+        internal static bool IsCustomAttributeDefinedHelper(
```

Comment:
Can this change be submitted as separate PR? It looks independent on the rest of the changes.

Reply:
Opened https://github.com/dotnet/runtime/pull/45621